src/components/RegionFlaggedContent.js

```diff
@@ -0,0 +1,43 @@
+import { useState } from 'react'
+import { div, h, li, span, table, tbody, tr, td, ul } from 'react-hyperscript-helpers'
+import { Link } from 'src/components/common'
+import { icon } from 'src/components/icons'
+import { TextCell, TooltipCell } from 'src/components/table'
+import * as Utils from 'src/libs/utils'
+import { withErrorReporting } from 'src/libs/error'
+import * as clipboard from 'clipboard-polyfill/text'
+
+
+const RegionFlaggedContent = ({ location, locationType, text: content, ...props }) => {
+
+
+  const [copied, setCopied] = useState()
+  const { flag, aka } = (locationType === 'multi-region' && location === 'US') ? { flag: '🇺🇸', aka: 'USA' } :
+    (location === 'EUROPE-NORTH1') ? { flag: '🇫🇮', aka: 'Finland' } :
+      { flag: undefined, aka: undefined }
```

Comment:
TOL. I worry about the tech debt of this block of code. The flags are nice and cute, but also now maintenance that we are creating for ourselves, that we have to remember to fix everytime we add a new region (which isn't in/near this code to remind us). I'd like a cleaner "failure" state that at least displays the text of the region. It might not be pretty, but would still display something to the user rather than nothing.

Reply:
+1

IMO we should display only the region, with `monospaced font styling`